alpaca/broker/models/funding.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
+from typing import Optional
+from uuid import UUID
+
+from ..enums import (
+    ACHRelationshipStatus,
+    BankAccountType,
+    BankStatus,
+    IdentifierType,
+    TransferType,
+    TransferStatus,
+    TransferDirection,
+    FeePaymentMethod,
+)
+from alpaca.common.models import ValidateBaseModel as BaseModel
+
+# TODO: Base funding class?
```

Comment:
All of the funding models (so far) include `id, account_id, created_at, updated_at`, so I was thinking about making a `BaseFundingModel` with these included, which the others would extend from. I am not sure if this is worth it though because it will hurt readability and may not be "future proof".

Reply:
Yeah without any common functionality between them I don't believe we need to anchor them together in that way and thus separated classes with similar fields is probably the best way forward